processing/src/main/java/org/apache/druid/java/util/common/guava/ParallelMergeCombiningSequence.java

```diff
@@ -475,20 +475,22 @@ private int computeNumTasks()
 
       final int computedNumParallelTasks = Math.max(computedOptimalParallelism, 1);
 
-      LOG.debug(
-          "Computed parallel tasks: [%s]; ForkJoinPool details - sequence parallelism: [%s] "
-          + "active threads: [%s] running threads: [%s] queued submissions: [%s] queued tasks: [%s] "
-          + "pool parallelism: [%s] pool size: [%s] steal count: [%s]",
-          computedNumParallelTasks,
-          parallelism,
-          getPool().getActiveThreadCount(),
-          runningThreadCount,
-          submissionCount,
-          getPool().getQueuedTaskCount(),
-          getPool().getParallelism(),
-          getPool().getPoolSize(),
-          getPool().getStealCount()
-      );
+      if (LOG.isDebugEnabled()) {
+        LOG.debug(
```

Comment:
Maybe also assign the result of `getPool` to a variable for simplicity and reuse it in the log line.

Reply:
extracted as variable done